test/fuzz/fuzzer.ml

```diff
@@ -69,6 +74,20 @@ let compare (module I1 : Interprets.INTERPRET)
 
 let check (module I1 : Interprets.INTERPRET) (module I2 : Interprets.INTERPRET)
   m =
+  (* Save the generated module *)
+  if Param.save_modules then (
```

Comment:
In the case of `if` it is more idiomatic to use `begin ... end` rather than `( ... )`.